pkgs/top-level/all-packages.nix

```diff
@@ -11556,6 +11556,8 @@ with pkgs;
   zabbix20 = callPackage ../servers/monitoring/zabbix/2.0.nix { };
   zabbix22 = callPackage ../servers/monitoring/zabbix/2.2.nix { };
 
+  zipkin = callPackage ../servers/monitoring/zipkin { };
```

Comment:
I wasn't sure what to put here: `zipkin` or `zipkin-server` ? Should it include a version number?